### DISCUSSION TITLE
Why TN in the fp8 gemm example?

### DISCUSSION THREAD

**Discussion by alecco** [NVIDIA/cutlass#1537] [Q&A] [answered]
The example 58_ada_fp8_gemm uses A transposed and B not transposed. I see the same is in cuBblasLt matmul for FP8. Why is this? Is this an instruction limitation?

Thanks

**Comment by alecco** [comment, marked as answer]
Seems it is.

**Reply by MARD1NO** [reply]
fp8 mma only support TN layout
